interface1.html

```diff
@@ -1074,9 +1074,9 @@ <h3>StrongStart</h3>
 
 					<div class="constraint-selection-table">
 						<div class="constraint-row">
-							<span><input type="checkbox" name="constraints" value="exhaust1">Exhaustivity</span>
+							<span><input type="checkbox" name="constraints" value="exhaustChild">Exhaustivity</span>
```

Comment:
We will actually want to have exhaustParent as the version of exhaustivity that's visible on the interface by default. 
exhaustChild should go in the "show more" section.